test/extended/util/client.go

```diff
@@ -345,6 +393,44 @@ func (c *CLI) TeardownProject() {
 	}
 }
 
+// SetupWorkSpace creates a new WorkSpace under the org workspace
+func (c *CLI) SetupWorkSpace() {
+	c.SetupWorkSpaceWithSpecificPath(c.orgServerURL)
```

Comment:
@Phaow does this create only universal workspace or any other type of workspace provided we give it a path ?

Reply:
Yeah, actually I just call `kubectl ws --server=XXX create XXX`, so it may create any path of workspace you want just need you have the rbac right.

Reply:
@Phaow when you raise  a new pr can you address the new changes in 0.7 into this function and change the comments as well ? I can merge this for now.